mlflow/utils/autologging/__init__.py

```diff
@@ -110,13 +110,10 @@ def _update_wrapper_extended(wrapper, wrapped):
     return updated_wrapper
 
 
-def wrap_patch(destination, name, patch, settings=None):
+def _wrap_patch(destination, name, patch, settings=None):
```

Comment:
I just confirmed on master branch that wrap_patch is not called outside of autologging_utils and test_autologging_utils files